src/main/app/claims/ccjModelConverter.ts

```diff
@@ -0,0 +1,66 @@
+import { DraftCCJ } from 'ccj/draft/DraftCCJ'
+import { PaidAmountOption } from 'ccj/form/models/yesNoOption'
+import { RepaymentPlan as RepaymentPlanForm } from 'ccj/form/models/repaymentPlan'
+import { PaymentType } from 'ccj/form/models/ccjPaymentOption'
+import {
+  CountyCourtJudgmentImmediatePayment,
+  CountyCourtJudgmentPaidByInstalments,
+  CountyCourtJudgmentPaidFullBySetDate
+} from 'claims/models/countyCourtJudgment'
+import { convertDefendantDetails } from 'claims/converters/defendantDetails'
+import { RepaymentPlan } from 'claims/models/replaymentPlan'
+import { PartyDetails } from 'forms/models/partyDetails'
+
+function convertRepaymentPlan (repaymentPlan: RepaymentPlanForm): RepaymentPlan {
+
+  if (repaymentPlan && repaymentPlan.remainingAmount !== undefined) {
+    return new RepaymentPlan(
+      repaymentPlan.remainingAmount,
+      repaymentPlan.firstPayment,
+      repaymentPlan.installmentAmount,
+      repaymentPlan.firstPaymentDate.asString(),
```

Comment:
why as string? shouldn't it be a moment?

Reply:
Well, I found some other places where we send string to API. Should we always use moment to transfer data?

Reply:
Depends on if the API can handle it.
The class that we use for handling the API data model should use `moment`